pyodide-build/pyodide_build/mkpkg.py

```diff
@@ -140,17 +176,19 @@ def success(msg):
     print(bcolors.OKBLUE + msg + bcolors.ENDC)
 
 
-def update_package(package: str, update_patched: bool = True):
+def update_package(
+    package: str, update_patched: bool = True, wheel: bool = True, sdist: bool = False
+):
 
     YAML = _import_ruamel_yaml()
-
     yaml = YAML()
 
     meta_path = PACKAGES_ROOT / package / "meta.yaml"
-    try:
-        yaml_content = parse_package_config(meta_path)
-    except:
+    if not meta_path.exists():
+        print(f"{meta_path} does not exist")
         sys.exit(0)
+    with open(meta_path, "rb") as fd:
+        yaml_content = yaml.load(fd)
```

Comment:
Slightly better to use the previous function, no, in case we change exactly how we parse/validate the .yaml files.

Reply:
The problem is `parse_package_config` doesn't use `ruamel.yaml` so if you use `parse_package_config` then `mkpkg update` will destroy comments.